src/examples/rover_steering_control/main.cpp

```diff
@@ -156,6 +156,23 @@ void control_attitude(const struct vehicle_attitude_setpoint_s *att_sp, const st
 		      struct actuator_controls_s *actuators)
 {
 
+  //double r,p,y,t;
+  
+
+  
+  //printf("vehicle_attitude_setpoint:\n");
+  //print values only when they change to get less spam in console
+  if((double)att_sp->roll_body > 0.0) { printf("att_sp->roll_body: %8.4f\n" , (double)att_sp->roll_body); }
```

Comment:
This code part has issues with types and with the range check - you have to check +- here. Plus we should not have commented out code in master or controllers printing to the console. Either guard it with a verbose or debug flag or please remove it entirely.